webpack-subresource-integrity/src/util.ts

```diff
@@ -241,15 +253,19 @@ export function replaceInSource(
   compiler: Compiler,
   source: sources.Source,
   path: string,
-  replacements: Iterable<[string, string]>
+  replacements: Map<string, string>
 ): sources.Source {
-  const oldSource = source.source();
+  const oldSource = source.source() as string;
```

Comment:
I'd prefer if we didn't cast here, as this might conceivably be a `Buffer`. Let's use `typeof` and bail out early if it's not a string?